openedx/features/redhouse_panel/api/v0/serializers.py

```diff
@@ -0,0 +1,6 @@
+from rest_framework import serializers
+
+
+class SiteSerializer(serializers.Serializer):
```

Comment:
Why not ModelSerializer?

Reply:
@danialmalik data is coming from two different models